libiocage/lib/Jail.py

```diff
@@ -390,7 +390,7 @@ def create(self, release_name):
 
         backend = None
 
-        is_basejail = self.config["type"] == "basejail"
+        is_basejail = self.config.get("basejail", False)
```

Comment:
self.config["basejail"] is already a boolean because it returns the parsed user input.

Reply:
Sure. It grabs a value from a dictionary key. If it doesn't exist it becomes the default value. On this case False. 

The equivalent of try;except KeyError;foo 

>

Reply:
This is so smart! Thanks for the clarification.